spec.bs

```diff
@@ -129,6 +129,20 @@ Note: This section is non-normative.
   a direct developer-to-provider API before granting entitlements.
 </div>
 
+## History of purchases ## {#history-of-purchases}
+
+<div class="example" id="History of purchases">
+  <xmp highlight="javascript">
+    const purchaseHistory = await digitalGoodsService.listPurchaseHistory();
+    for (p of purchaseHistory) {
+      DisplayPreviousPurchase(p.itemId);
+    }
+  </xmp>
+
+  The {{DigitalGoodsService/listPurchaseHistory()}} method allows a client
+  to list user's previous purchases.
```

Comment:
This is actually "the latest purchase for each item type ever purchased by the user. May include expired or consumed purchases." (according to our mojo definition).
There should probably be a caveat here that stores might not keep such history, in which case it would return the same thing as ListPurchases.